web/templates/left_sidebar.hbs

```diff
@@ -136,8 +136,8 @@
     <div id="direct_messages_sticky_header" class="direct-messages-container zoom-out hidden-for-spectators">
         <div id="direct_messages_section">
             <div id="direct_messages_section_header" class="zoom-out zoom-in-sticky">
-                <i id="toggle_direct_messages_section_icon" class="fa fa-sm fa-caret-down dm-tooltip-target toggle_private_messages_section zoom-in-hide" aria-hidden="true" tabindex="0" role="button"></i>
-                <h4 class="left-sidebar-title toggle_private_messages_section"><span class="dm-tooltip-target">{{t 'DIRECT MESSAGES' }}</span></h4>
```

Comment:
Is this unused? I'm seeing two JavaScript references to `toggle_private_messages_section`--one in `click_handlers.js`, and one in `pm_list.ts`. Please check all of your removals here with `git grep`.

Reply:
All the relevant code had been removed for both the removal commits, the two references you are seeing are for a function with the same name as the class.

The first two commits only remove css but their respective id's are still in use either in javascript or other css properties.

Reply:
This was completed in #30501.